src/console/Program.cs

```diff
@@ -112,6 +161,15 @@ public static ServiceProvider GetRequiredServiceProvider(IConfiguration config,
             }
         }
 
+        public static void ConfigureLogging(IConfiguration config, IServiceCollection serviceCollection)
+        {
+            var instrumentationKey = config.GetSection("APPINSIGHTS_INSTRUMENTATIONKEY").Value;
```

Comment:
>APPINSIGHTS_INSTRUMENTATIONKEY [](start = 56, length = 30)

What happens when this is null?

Reply:
Updated. Now it will continue and just not log to app insights if null/empty.